README.md

```diff
@@ -132,6 +132,20 @@ Future<void> testExecutable(FutureOr<void> Function() testMain) async {
 
 As an alternative you can use [Alchemist](https://pub.dev/packages/alchemist).
 
+Also, you can configure `AdaptiveTestConfiguration` singleton to skip tests instead of throwing if they are run on an unintended platform.
+```dart
+Future<void> testExecutable(FutureOr<void> Function() testMain) async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  AdaptiveTestConfiguration.instance
+    ..setSkippedTestPlatform(TargetPlatform.macOS)
```

Comment:
setSkippedTestPlatform function doesn't seem to exist in this PR. 
do you mean `..setEnforcedTestPlatform(TargetPlatform.macOS)`?

Reply:
Thanks for noticing.
Fixed ✅